Taskfile.yml

```diff
@@ -0,0 +1,46 @@
+version: '2'
+
+vars:
+  INSTALL_DIR: "{{env \"HOME\"}}/cmswww"
+  DATA_DIR: "{{env \"HOME\"}}/.cmswww/data/testnet3/cockroachdb/data"
+
+tasks:
+  dataload:
+    desc: Runs cmswwwdataload to fill out database with some sample data
+    cmds:
+      - cmswwwdataload --verbose --deletedata
+
+  init_cdb:
+    desc: Deletes (if necessary) and (re)creates database ready for use
+    cmds:
+      - task: quit_cdb
+      - task: reset_cdb
+      - rm -rf ~/.cmswww/data/testnet3/cockroachdb
```

Comment:
Could you make "testnet3" a variable too?

Reply:
How does this look?

I also added a new task for vendoring the project.

Reply:
Looks good to me, is it ready to go in?

Reply:
Yep.  If there's any issues I'll submit a new PR, as I'll be using it.